### PR TITLE
18f purchase form

### PR DESCRIPTION
**Pull request by nickbristow**
Work In progress, 

Please hold off merging.

### CODE REVIEW

**Comment by afeld** [comment]
Screenshot or it didn't happen :wink:

**Comment by nickbristow** [comment]
c2
@afeld

File: app/models/gsa18f/proposal_form.rb [hunk 1]
```diff
@@ -0,0 +1,124 @@
+#module Ncr
+module Gsa18f
+  DATA = YAML.load_file("#{Rails.root}/config/data/18f.yaml")
+
+  class ProposalForm
+    include SimpleFormObject
+
+    EXPENSE_TYPES = %w(BA61 BA80)
```
Comment (phirefly) [inline app/models/gsa18f/proposal_form.rb]: These are NCR specific and can be removed

File: app/models/gsa18f/proposal_form.rb [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+    URGENCY = DATA['URGENCY']
+    OFFICES = DATA['OFFICES']
+    attribute :origin, :string
```
Comment (phirefly) [inline app/models/gsa18f/proposal_form.rb]: Similarly, we can remove NCR-specific attributes

File: app/models/gsa18f/proposal_form.rb [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+
+    def finance_approver_email
+      ENV['NCR_FINANCE_APPROVER_EMAIL'] || 'communicart.ofm.approver@gmail.com'
+    end
+
+    def requires_finance_approval?
+      self.expense_type == 'BA61'
+    end
+
+    def approver_emails
```
Comment (phirefly) [inline app/models/gsa18f/proposal_form.rb]: In the 18F case, I'm thinking we can just make Ric the approver_email and not even require the requester to type in his email address. This means that 1) removal from the form, and 2) updating `approver_emails` to be `['rics@email_address']`

Comment (phirefly) [inline app/models/gsa18f/proposal_form.rb]: Another option is to not hard code his email and keep it in the yaml. Thoughts?

Comment (cmc333333) [inline app/models/gsa18f/proposal_form.rb]: Ultimately, I'd put it in the db. In the meanwhile, I'd just add an environment variable for 18F_APPROVER_EMAIL

**Comment by phirefly** [comment]
I'm good with that

**Pull request closed without merging**